tcms/testplans/static/testplans/js/get.js

```diff
@@ -253,6 +253,55 @@ function toolbarEvents() {
 
         sortTestCases();
     });
+
+    $('.js-toolbar-priority').click(function(ev) {
+        let selectedCases = getSelectedTestCases();
+
+        if (!selectedCases.length) {
+            alert($('#test_plan_pk').data('trans-no-testcases-selected'));
+            return;
+        }
+
+        for (let i = 0; i < selectedCases.length; i++) {
+            let testCaseId = selectedCases[i];
+            jsonRPC('TestCase.update', [testCaseId, {'priority': ev.target.dataset.id}], function() {
+                const testCaseRow = $(`.js-testcase-row[data-testcase-pk=${testCaseId}]`);
+                testCaseRow.fadeOut(fadeAnimationTime, function() {
+                    testCaseRow.find('.js-test-case-priority').html(ev.target.innerText);
+                }).fadeIn(fadeAnimationTime);
```

Comment:
in the `refactor_testrun_ui` I extracted this in `utils.js` - https://github.com/kiwitcms/Kiwi/blob/refactor_testrun_ui/tcms/static/js/utils.js#L169

maybe it's a good idea to do the same here, or merge my change into `master`, so that we don't have to walk through your code later and refactor all of those to use my function

Reply:
https://github.com/kiwitcms/Kiwi/pull/1951

Reply:
I will use it.

Reply:
The refactor_testplan_ui branch has been rebased to master so you already have access to the animate function. Should we wait for using it in this PR ?

Reply:
No. I use it in https://github.com/kiwitcms/Kiwi/pull/1953.